packages/shared/lib/services/sync/manager.service.ts

```diff
@@ -449,6 +466,7 @@ export class SyncManagerService {
             frequency,
             latestResult: latestJob?.result,
             latestExecutionStatus: latestJob?.status,
+            recordCount,
             ...(includeJobStatus ? { jobStatus: latestJob?.status as SyncStatus } : {})
         } as ReportedSyncJobStatus;
```

Comment:
type was not updated. 
not your fault but the usage of `as` here is problematic (as always). I just tried to replace it with `satisfies` and there are more errors :/

Reply:
good catch. I removed the casting and fixed the type. There might be underlying issue with the type of some attributes like the `type` one but I didn't want to change the behavior in this PR